R/predict.R

```diff
@@ -51,10 +51,15 @@ predict.WrappedModel = function(object, task, newdata, subset = NULL, ...) {
     assertClass(task, classes = "Task")
     size = getTaskSize(task)
   } else {
-    assertDataFrame(newdata, min.rows = 1L)
```

Comment:
Please don't delete that. Might not protect against a lot of things but at least some. Just put it after `if(xts...) { } else { }`.